dds/DCPS/XTypes/DynamicData.cpp

```diff
@@ -2260,7 +2299,6 @@ bool DynamicData::skip_all()
           break;
         }
       }
-      release_chains();
```

Comment:
Why are these calls removed? They must be kept to release the intermediate message block chains it created. Otherwise, there can be memory leak.